app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -17,7 +18,24 @@
 <!-- Persistent URL block end -->
 <%= presenter.attribute_to_html(:date_modified, label: t('hyrax.base.show.last_modified'), html_dl: true) %>
 <%= facet_renderer.new(:content_genre, presenter.solr_document['content_genre_ssi'], {}).render_dl_row  %>
-<%= presenter.attribute_to_html(:creator, render_as: :faceted, html_dl: true) %>
+<dt>Creator</dt>
```

Comment:
@alexBLR Can you abstract this code away into a partial and/or helper ([like these](https://github.com/samvera/hyrax/blob/main/app/helpers/hyrax/hyrax_helper_behavior.rb))?

Reply:
Absolutely